connectors/perl/src/orders/parser.cc

```diff
@@ -54,27 +55,30 @@ void parser::execute(const std::string& cmd) {
   char* ptr(nullptr);
   unsigned long long cmd_id(strtoull(cmd.c_str(), &ptr, 10));
   if (!cmd_id || *ptr)
-    throw basic_error() << "invalid execution request received:"
-                           " bad command ID ("
-                        << cmd.c_str() << ")";
+    throw msg_fmt(
+        "invalid execution request received:"
+        " bad command ID ({})",
+        cmd.c_str());
```

Comment:
string badly formatted and cmd.c_str() can be replaced by cmd

Reply:
will fix